<?xml version="1.0"?>
<robot name="Jarvis">
  <link name="base_link">
     <visual>
		<geometry>
			<cylinder length="0.01" radius="0.00"/>

      	</geometry>
       <origin rpy=" 0 0 0" xyz="0 0 0"/>
       <material name="white">
            <color rgba="1 1 1 1"/>
       </material>
     </visual>
   </link>
  
  <link name="link1">
     <visual>
		<geometry>
			<cylinder length="1" radius="0.04"/>

      	</geometry>
       <origin rpy=" 0 1.57 0" xyz="0.5 0 0"/>
       <material name="white">
            <color rgba="1 1 1 1"/>
       </material>
     </visual>
   </link>
   
     <link name="link2">
     <visual>
		<geometry>
			<cylinder length="1" radius="0.04"/>

      	</geometry>
       <origin rpy=" 0 1.57 0" xyz="0.5 0 0"/>
       <material name="blue">
            <color rgb="13 111 19"/>
       </material>
     </visual>
   </link>
   
   <link name="link3">
                <origin rpy=" 0 1.57 0" xyz="1 0 0"/>
   </link>
   
   
  <joint name="joint1" type="continuous">
        <parent link="base_link"/>
	    <child link = "link1"/>
	    <origin xyz="0 0 0" rpy="0 0 0" />
  	    <axis xyz="0 0 1" />

	
 	    <calibration rising="0.0"/>
  	    <dynamics damping="0.0" friction="0.0"/>
   </joint>
   
     <joint name="joint2" type="continuous">
        <parent link="link1"/>
	    <child link = "link2"/>
	    <origin xyz="1 0 0" rpy="0 0 0" />
  	    <axis xyz="0 0 1" />
	
 	    <calibration rising="0.0"/>
  	    <dynamics damping="0.0" friction="0.0"/>
   </joint>
   
    <joint name="joint3" type="continuous">
        <parent link="link2"/>
	    <child link = "link3"/>
	    <origin xyz="1 0 0" rpy="0 0 0" />
  	    <axis xyz="0 0 1" />
	
 	    <calibration rising="0.0"/>
  	    <dynamics damping="0.0" friction="0.0"/>
   </joint>
   
   

</robot>
